packages/opentelemetry/src/propagator.ts

```diff
@@ -91,9 +88,14 @@ export class SentryPropagator extends W3CBaggagePropagator {
 
     const activeSpan = trace.getSpan(context);
     const url = activeSpan && getCurrentURL(activeSpan);
+    // We only want to check the URL against tracePropagationTargets for outgoing request spans
+    // In other cases, we always inject the trace data
+    const shouldCheckUrl =
```

Comment:
Without this, injecting metadata e.g. in remix (but possibly other places!) would fail, because in this scenario the active span may be e.g. the http.server span, which has a `http.url` attribute that we would check and which may fail to check against `tracePropagationTargets`, disabling injection and thus not generating any meta tags.

Now, we only check against the http.url if the span is a http.client span.